### PR TITLE
fix: update table_schema when instantiating already existing BigQueryVectorStore

### PR DESCRIPTION
**Pull request by Freezaa9**
Following this issue:
fix: https://github.com/langchain-ai/langchain-google/issues/426

Update table_schema when table schema already exist.

Otherwise it is only done during add_text

### CODE REVIEW

**Comment by Freezaa9** [comment]
thanks @isahers1
lint done

File: libs/community/tests/integration_tests/test_bigquery_vector_search.py [hunk 1]
```diff
@@ -106,3 +113,27 @@ def test_get_doc_by_filter(self, store: BigQueryVectorSearch) -> None:
         assert "fruit" in kinds
         assert "treat" not in kinds
         assert "planet" not in kinds
+
+    @pytest.mark.skip(reason="investigating")
```
Comment (lkuligin) [inline libs/community/tests/integration_tests/test_bigquery_vector_search.py:117]: why is it marked as skipped? is this new test failing?

Comment (Freezaa9) [reply libs/community/tests/integration_tests/test_bigquery_vector_search.py]: Hi,
My bad I edited the wrong test file. It was the tets file for the outdated BigQueryVectorSearch.
I added the test in test_feature_store_bq_vectorstore.py and test are passing locally

**Comment by lkuligin** [comment]
@Freezaa9 please, feel free to take a look at https://github.com/langchain-ai/langchain-google?tab=readme-ov-file#code-formatting (how to run linter locally), it might help to you to move faster

**Comment by Freezaa9** [comment]
> @Freezaa9 please, feel free to take a look at https://github.com/langchain-ai/langchain-google?tab=readme-ov-file#code-formatting (how to run linter locally), it might help to you to move faster

done thanks !

**Comment by lkuligin** [comment]
> > @Freezaa9 please, feel free to take a look at https://github.com/langchain-ai/langchain-google?tab=readme-ov-file#code-formatting (how to run linter locally), it might help to you to move faster
> 
> done thanks !
> 
> EDIT: And I don't know what to do for codespell fail and lint reformating for files unrelated to this PR. Codespell: `Error: ./PULL_REQUEST_TEMPLATE.md:76: someting ==> something` Lint:
> 
> ```
> \langchain-google\libs\community\langchain_google_community\bq_storage_vectorstores\_base.py
> \langchain_google_community\bq_storage_vectorstores\featurestore.py
> \langchain-google\libs\community\tests\unit_tests\test_googlesearch_api.py
> ```
> 
> Thanks

just fix the typo, please. not sure how we didn't catch it, sorry about it.

**Comment by lkuligin** [comment]
@eliasecchig

**Review by lkuligin — APPROVED** [review]

**Review by eliasecchig — APPROVED** [review]

**Pull request merged**